docs/getting-started/accessing-resources/getting-access-to-data.md

```diff
@@ -120,3 +130,95 @@ This will download test data and direct `data/current` symlink to the test data
+Results are available at specific tagged releases. <!-- STUB_LINK to openscpca-nf docs -->
+
+We briefly cover some of the most common use cases below, but we encourage you to review all the options available to you.
+
+You can list all the options for `download-results.py` by running the following from the root directory of the repository:
+
+```sh
+./download-results.py --help
+```
+
+!!! tip "No `--profile` necessary on Lightsail for Research"
+    Omit `--profile openscpca` from the commands below if you're using Lightsail for Research.
+
+Example script usage below assumes you have created a profile called `openscpca` when [configuring the AWS CLI](../../technical-setup/environment-setup/configure-aws-cli.md).
+
+To download all results for one or more modules, use the `--modules` option as follows:
+
+```sh
+# Get results from a single module
+./download-results.py --modules module-name --profile openscpca
+
+# Get results from several modules
+./download-results.py --modules first-module,second-module --profile openscpca
+```
+
+To list all available modules for which you can download results, you can use the `--list-modules` option as follows.
+This will list all modules available in, by default, the current release:
+
+```sh
+./download-results.py --list-modules --profile openscpca
+```
+
+To review what result files would be downloaded without performing the download yet, you can use the `--dryrun` option as follows:
+
+```sh
+./download-data.py --modules module-name --dryrun --profile openscpca
+```
+
+If you wish to only working with a subset of a module's results, you can use the `--projects` or `--samples` options to download select samples (note: these options are mutually exclusive).
+Note that whether this subsetting is possible will depend on whether that module's results have been structured in a manner that reflects this organization. PHRASING HELP PLEASE!
+
+#### Download result data structure
+
+By default, results will download into the `data` directory in your local copy of the `OpenScPCA-analysis` repository, but this can be customized with the `--data-dir` option.
+
+Result downloads will generally follow this structure:
+
+```sh
+data
+├── {Release}
+│       └── results
+│          └── {Module name}
+│              └── {Result files}
+└── current -> {Release}
+```
+
+Both result files and ScPCA data files follow the same release schedule.
+Therefore, if you have downloaded both data and workflow results, all files will be present in the `data` directory with this overall structure:
+
+```sh
+data
+├── {Release}
+│       ├── {Project ID}
+│       │   └── {Sample ID}
+│       │       └── {Library files}
+│       ├── bulk_metadata.tsv (if applicable)
+│       ├── bulk_quant.tsv (if applicable)
+│       ├── single_cell_metadata.tsv
```

Comment:
The bulk and metadata should be nested under Project ID. Noting this is incorrect in the data download description as well.

Reply:
https://github.com/AlexsLemonade/OpenScPCA-analysis/pull/503/commits/efe5cd4f7dea45e38dc0409a3b4c2fa628390875